### ISSUE TITLE
confusing config property name `noteKeywords`

### ISSUE THREAD

**Issue by grubyak** [conventional-changelog/conventional-changelog#354]
1. `release > generateNotes > parserOpts > noteKeywords` used to indicate important notes (sets release type be `major`) is not self-describing - for example `breakingChangeKeywords` could be used instead

2. documentation could be adjusted to inform that given keywords must appear at the beginning of the comment line

**Comment by GirasolXimena** [comment]
I agree! Needed this property for a project and only found out about it months later due to the confusing

**Comment by dangreen** [comment]
Thanks for the suggestion. noteKeywords is a general parser option for release notes; it is not limited to breaking changes, so a name such as breakingChangeKeywords would be misleading. The option has been part of the public configuration for a long time, and renaming it would create unnecessary churn and compatibility cost. We do not plan to rename it. Closing as not planned.
